code/modules/projectiles/guns/magic.dm

```diff
@@ -16,7 +16,7 @@
 	var/no_den_usage
 	origin_tech = null
 	clumsy_check = 0
-	trigger_guard = 0
+	trigger_guard = -1 // Has no trigger at all, uses magic instead

```

Comment:
please make this a define

#define TRIGGER_GUARD_MAGIC    -1
#define TRIGGER_GUARD_NORMAL  1
#define TRIGGER_GURAN_NONE       0